tests/controllers/send_ctrl_spec.coffee

```diff
@@ -66,6 +77,14 @@ describe "SendCtrl", ->
 
     return
 
+  beforeEach ->
+    $httpBackend.expectGET('http://service-dynamic-fee.prod.blockchain.co.uk/fees')
```

Comment:
This needs to be dynamic, if only because production is HTTPS. Easiest way is to use grunt magic similar to how $rootScope.rootURL is set. I'll look into that.